WordPress/src/main/res/layout/add_new_site_dialog_item.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/text"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:minHeight="?android:attr/listPreferredItemHeightSmall"
+    android:textAppearance="?android:attr/textAppearanceListItem"
+    android:textColor="?android:attr/textColorAlertDialogListItem"
+    android:drawableStart="@drawable/ic_add_outline_grey_dark_24dp"
```

Comment:
The `drawableStart` attribute does not support vector drawables.  It will cause crashes on pre-Lollipop devices.  We'll have to replace the vector with a raster drawable for this view.

Reply:
Just wanted to note that you can use a vector if you assign it at runtime. Here's an example from `PluginDetailActivity`:

```
Drawable rightDrawable = AppCompatResources.getDrawable(this, R.drawable.ic_info_outline_grey_dark_18dp);
mVersionTopTextView.setCompoundDrawablesWithIntrinsicBounds(null, null, rightDrawable, null);
```

Reply:
Nice!  I didn't know that.  Thanks for the tip, @nbradbury!

Reply:
Nice catch! I utilized @nbradbury 's tip (thanks Nick!) in 0d3a759.